### PR TITLE
feat: cubejs integration 

### PR DESCRIPTION
**Pull request by guptamanish07**
cubejs integration with frappe

### CODE REVIEW

File: bloomstack_core/bloomstack_core/page/admin_insights/admin_insights.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import frappe
+
+
+@frappe.whitelist()
+def get_cubejs_host():
+    print("function is calling from home")
```
Comment (hrwX) [inline bloomstack_core/bloomstack_core/page/admin_insights/admin_insights.py]: - Remove the print statements.
- Also how are you planning to put the credentials in the site config in the first place ?

File: bloomstack_core/bloomstack_core/page/admin_insights/admin_insights.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			'assets/bloomstack_core/js/min/admin_insights.min.js'
+		];
+		frappe.call({
+			method:
+				"bloomstack_core.bloomstack_core.page.admin_insights.admin_insights.get_cubejs_host",
+			callback: (r) => {
+				var cube_js_host = r.message.cube_js_host;
+				var cube_js_secret = r.message.cube_js_secret;
+				localStorage.setItem("cube_js_host", cube_js_host);
+				localStorage.setItem("cube_js_secret", cube_js_secret)
```
Comment (hrwX) [inline bloomstack_core/bloomstack_core/page/admin_insights/admin_insights.js]: - do not store the credentials in localstorage

**Review by hrwX — CHANGES_REQUESTED** [review]

File: config.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+// export const CUBE_JS_HOST="http://localhost:4000";
+// export const CUBE_JS_SECRET = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE2MDAxNjkxOTMsImV4cCI6MTYwMjc2MTE5M30.M09WQCwOHkt-ZHhtdNk0Nk7IGi2krlEH-53y90RO-G4";
+export const CUBE_JS_HOST = "https://cubejs.bloomstack.io";
+export const CUBE_JS_SECRET = "4ee295f55c613cfef9b4bb48eeeea4db7d6c9ce0fb25ebd66b7f4e6ba9409a92b85bdfbc847dec791dda588e0810246023d98e8241906f53d87dd13892eedbd4";
```
Comment (hrwX) [inline config.js]:
```suggestion
export const CUBE_JS_HOST = "https://cubejs.bloomstack.io";
export const CUBE_JS_SECRET = "XXXX";
```

**Review by hrwX — CHANGES_REQUESTED** [review]

**Comment by hrwX** [comment]
@07061999 Why are there `min.js` files in the PR ?? They should be built and stored in sites folder.

**Comment by vigilantee** [comment]
@07061999 resolve codacy issues.

**Review by vigilantee — APPROVED** [review]

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**